openstack/cinder/templates/backup_deployment.yaml

```diff
@@ -35,7 +35,7 @@ template: |{{`
         hostname: cinder-volume-backup-vmware-{{ name }}
         containers:
         - name: cinder-volume-backup-vmware-{{ name }}
-          image: `}}{{.Values.global.imageRegistry}}/{{.Values.global.image_namespace}}/loci-cinder:{{.Values.imageVersionCinderVolumeBackup | default .Values.imageVersion | required "Please set cinder.imageVersion or similar" }}{{`
+          image: `}}{{.Values.global.rgistry}}/loci-cinder:{{.Values.imageVersionCinderVolumeBackup | default .Values.imageVersion | required "Please set cinder.imageVersion or similar" }}{{`
```

Comment:
Can you change this to look like the other image: lines, i.e. add the `required`, too?

Reply:
should be fixed.